src/main/java/core/screens/HUD/HudManager.java

```diff
@@ -0,0 +1,108 @@
+package core.screens.HUD;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.Disposable;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+
+public class HudManager implements Disposable {
+
+    public Stage stage;
+    private Viewport viewport;
+
+    //score && time tracking variables
+    private Integer worldTimer;
+    private float timeCount;
+    private static Integer score;
+    private boolean timeUp;
+
+    //Scene2D Widgets
+    private Label countdownLabel, timeLabel, linkLabel;
+    private static Label scoreLabel;
+
+    public HudManager(SpriteBatch sb) {
+        //define tracking variables
+        worldTimer = 250;
+        timeCount = 0;
+        score = 0;
```

Comment:
add TODO comment to get these data from the LevelManager

Reply:
done!